### ISSUE TITLE
[.NET 8.0] Build crash System.Security.Permissions missing

### ISSUE THREAD

**Issue by Px7-941** [dansiegel/Mobile.BuildTools#334]
# Description

Build crashes with "Could not load file or assembly System.Security.Permissions" .

https://learn.microsoft.com/en-us/dotnet/core/compatibility/extensions/8.0/configurationmanager-package
https://learn.microsoft.com/en-us/dotnet/core/compatibility/8.0#extensions

## Reproduction Steps

1. Install https://github.com/dotnet/core/blob/main/release-notes/8.0/preview/8.0.0-rc.2.md
2. Create .NET 8.0 console project
3. Setup appsettings feature from Mobile.BuildTools
4. Build

## Expected Behavior

Generate app settings class (https://mobilebuildtools.com/config/appsettings/).

## Actual Behavior

```
/Users/runner/.nuget/packages/mobile.buildtools/2.0.245/build/Mobile.BuildTools.targets(37,5): 
error MSB4018: The "LocateBuildToolsConfigTask" task failed unexpectedly. 
[/Users/runner/work/mobile-app/mobile-app/src/Mobile.Business/Mobile.Business.csproj::TargetFramework=net8.0]

/Users/runner/.nuget/packages/mobile.buildtools/2.0.245/build/Mobile.BuildTools.targets(37,5): 
error MSB4018: System.IO.FileNotFoundException: 
Could not load file or assembly 'System.Security.Permissions, Version=0.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. 
The system cannot find the file specified. 
[/Users/runner/work/mobile-app/mobile-app/src/Mobile.Business/Mobile.Business.csproj::TargetFramework=net8.0]
```

## Environment

- OS: [OSX]
- IDE: [VSMac|CLI|Rider]
- BuildTools Version: [2.0.245]
- IDE Version: [-]
- Build Target Framework: [MAUI-8.0|NET-8.0]

## Reproduction App

[ConsoleApp1.zip](https://github.com/dansiegel/Mobile.BuildTools/files/12926979/ConsoleApp1.zip)

**Comment by robfrancis** [comment]
Bump, with the release and installation of .net 8, a .net 7 project now fails with this error. Is there a release/pre-release coming for this?

Thanks.

**Comment by AdamDiament** [comment]
I have the same issue building a MAUI project, on the released version of net8 sdk (8.0.100) when building an android app on azure dev ops. Locally, it builds fine.

I tried adding a globals.json at the root of my project file with these contents, it doesn't help.

```json
    {
    "sdk": {
      "version": "8.0.100",
      "rollForward": "latestFeature"
    }
  }
```

**Comment by Px7-941** [comment]
Internally we use our own customized Nuget build with the System.Text.Json migration. 

https://github.com/dansiegel/Mobile.BuildTools/pull/335

**Comment by AdamDiament** [comment]
> Internally we use our own customized Nuget build with the System.Text.Json migration.
> 
> #335

@Px7-941 that sounds excellent. Is this customised nuget package something you could share at all?

**Comment by dansiegel** [comment]
> Internally we use our own customized Nuget build with the System.Text.Json migration.
> 
> #335

You know I do take PRs

**Comment by Px7-941** [comment]
@dansiegel you can check the PR. Your main/master did not work as our base. We used your last release branch.

**Comment by AdamDiament** [comment]
@Px7-941 thanks - would you be willing to share your local nuget package with me in the meantime please?

**Comment by Px7-941** [comment]
@AdamDiament here is our own Nuget, we only tested the appsettings generation feature. GitHub blocks the ".nupkg" extension, remove the .zip extension. 
[mobile.buildtools.2.0.39-g3081c6739d.nupkg.zip](https://github.com/dansiegel/Mobile.BuildTools/files/13672603/mobile.buildtools.2.0.39-g3081c6739d.nupkg.zip)

**Comment by AdamDiament** [comment]
> @AdamDiament here is our own Nuget, we only tested the appsettings generation feature. GitHub blocks the ".nupkg" extension, remove the .zip extension.
> [mobile.buildtools.2.0.39-g3081c6739d.nupkg.zip](https://github.com/dansiegel/Mobile.BuildTools/files/13672603/mobile.buildtools.2.0.39-g3081c6739d.nupkg.zip)

Thanks very much @Px7-941, it works like a dream! We only use the AppSettings feature in any case. 

@dansiegel thanks again for an amazing project

**Comment by dansiegel** [comment]
This was fixed with the release of the 2.1 preview. Be sure trying to generate the AppSettings that you use the new AppSettings Package. Other features in 2.1 aren't fully tested yet as the MAUI Team has broken a bunch of things.
https://www.nuget.org/packages/Mobile.BuildTools.AppSettings
